src/attached_probe.cpp

```diff
@@ -1020,11 +1016,26 @@ void AttachedProbe::attach_watchpoint(int pid, const std::string& mode)
   // Generate a notification every 1 event; we care about every event
   attr.sample_period = 1;
 
-  int perf_event_fd = bpf_attach_perf_event_raw(progfd_, &attr, pid, -1, -1, 0);
-  if (perf_event_fd < 0)
-    throw std::runtime_error("Error attaching probe: " + probe_.name);
+  std::vector<int> cpus;
+  if (pid >= 1)
+  {
+    cpus = std::vector<int>(1, -1);
```

Comment:
Nit: bit more clear with this instead:
```suggestion
    cpus = { -1 };
```

Reply:
done